e2e/src/tests/fileBrowserPage.test.js

```diff
@@ -0,0 +1,21 @@
+import { vBtn, vListItemSubtitle  } from 'jest-puppeteer-vuetify';
+import {
+    waitForRequestsToFinish,
+  } from '../util';
+
+describe('file browser page', () => {
+    it('view file browser', async () => {
+
+        await expect(page).toClickXPath(vBtn('Public Dandisets'));
+        await waitForRequestsToFinish();
+
+        // Assert that the page has loaded with a list of dandisets
+        await expect(page).toClickXPath(vListItemSubtitle('DANDI:000001'));
+        await expect(page).toClickXPath(vBtn('Files'));
+        await waitForRequestsToFinish();
+
+        // Ensure that the page has loaded with a folder
+        await expect(page).toMatch('foo');
```

Comment:
I feel as this is not ideal since the name for the folder could change. Preferably I would select the element with js but that would require us to install `jsdom` for jest and I don't know if that is wise for just this test.

Reply:
How does `.toMatch()` work on the `page` object? Can you use the `page` object's [`$()` method](https://pptr.dev/api/puppeteer.page._) to select the element?

You would still have the problem of hard-coding the name of the folder, but I don't think that's a big deal. If we ever change the folder name in the test dandisets, this test will fail and we'll just fix it.

Reply:
`.toMatch()` is just regex, you can't use jquery with it:
https://jestjs.io/docs/expect#tomatchregexp--string

To use js/jquery we would need to set the jest environment to `jsdom` which does not come with jest out of the box and would be an additional package:
https://jestjs.io/docs/configuration#testenvironment-string
https://www.npmjs.com/package/jest-environment-jsdom

In that case it would be something along the lines of `Page.(document.getElementsByCLassName('v-list')`

Reply:
After a bit more searching its likely to complete this using xpath ie `path.(//div[contains(@class,'v-list'))`. I'll give it a shot.

Reply:
implemented in 1258b94bb37a266cc6262f55b302201715c28300